### PR TITLE
[Python] Add note for using TF2 models with TensorFlow Hub

### PR DESCRIPTION
**Pull request by riteshghorse**
Added a note for using `TFModelHandler` with TF2 saved model format.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by riteshghorse** [comment]
R: @rszper

File: examples/notebooks/beam-ml/run_inference_with_tensorflow_hub.ipynb [hunk 1]
```diff
@@ -104,7 +104,10 @@
       "cell_type": "markdown",
       "source": [
         "## Use TensorFlow Hub's trained model URL\n",
-        "To use TensorFlow Hub's trained model URL, pass the model URL to the `model_uri` field of `TFModelHandler` class."
+        "To use TensorFlow Hub's trained model URL, pass the model URL to the `model_uri` field of `TFModelHandler` class.\n",
+        "\n",
+        "**Note:** Only models saved with [TF2 format](https://www.tensorflow.org/tutorials/keras/save_and_load#save_the_entire_model) are compatible with `TFModelHandler`.\n",
```
Comment (rszper) [inline examples/notebooks/beam-ml/run_inference_with_tensorflow_hub.ipynb:109]:
```suggestion
        "**Note:** Only models that you save in the [TF2 format](https://www.tensorflow.org/tutorials/keras/save_and_load#save_the_entire_model) are compatible with `TFModelHandler`.\n",
```

File: examples/notebooks/beam-ml/run_inference_with_tensorflow_hub.ipynb [hunk 2]
```diff
@@ -104,7 +104,10 @@
...
+        "Checkout [this](https://tfhub.dev/s?subtype=module,placeholder&tf-version=tf2) for TF2 models on TensorFlow Hub."
```
Comment (rszper) [inline examples/notebooks/beam-ml/run_inference_with_tensorflow_hub.ipynb:110]:
```suggestion
        "To see TF2 models, go to the [TF2 section of the TensorFlow Hub](https://tfhub.dev/s?subtype=module,placeholder&tf-version=tf2)."
```

**Review by rszper — APPROVED** [review]

**Comment by rszper** [comment]
LGTM

**Pull request merged**